src/elements/ExpiryIndicator.stories.ts

```diff
@@ -0,0 +1,45 @@
+import {fn} from '@storybook/test';
+import type {Meta, StoryObj} from '@storybook/vue3';
+
+import BubbleSelect from "@/elements/BubbleSelect.vue";
+import type {SelectOption} from "@/models";
+import ExpiryIndicator from "@/elements/ExpiryIndicator.vue";
+import { ExpiryUnitTypes } from "@/definitions";
+
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories
+const meta = {
+  title: "Elements/ExpiryIndicator",
+  component: ExpiryIndicator,
+  // This component will have an automatically generated docsPage entry: https://storybook.js.org/docs/writing-docs/autodocs
+  tags: ["autodocs"],
+  args: {
+    timer: 10,
+    warn: 5,
+    unit: ExpiryUnitTypes.Days,
+  },
+} satisfies Meta<typeof BubbleSelect>;
```

Comment:
This might be a copy paste error. The type has to be `typeof ExpiryIndicator` not BubbleSelect if I'm seeing that right. Also is there any reason to not explicitly add the type here?

```suggestion
const meta: Meta<typeof ExpiryIndicator> = {
  title: "Elements/ExpiryIndicator",
  component: ExpiryIndicator,
  // This component will have an automatically generated docsPage entry: https://storybook.js.org/docs/writing-docs/autodocs
  tags: ["autodocs"],
  args: {
    timer: 10,
    warn: 5,
    unit: ExpiryUnitTypes.Days,
  },
};
```

Reply:
Copy paste error for sure. Also not entirely sure about setting the type, I've not written typescript or vue before and have been trying to figure it out as I go and run into issues 😅